.sonata-project.yml

```diff
@@ -24,6 +24,10 @@ labels:
   - name: help wanted
     color: '0e8a16'
 
+files:
+  - doc/CONTRIBUTING.md:CONTRIBUTING.md
+  - tools/editorconfig/.editorconfig:.editorconfig
```

Comment:
Maybe we should keep it simple and have only one tree, matching exactly the destination tree. That way we may simply use a recursive copy.